miracl/flow/miracl_workflow_ace_interface.py

```diff
@@ -917,6 +929,55 @@ def _clear_seg_folders(seg_folder: Path):
         seg_folder.mkdir(parents=True, exist_ok=True)
 
 
+class ConversionChecker:
+    """Class for checking if each subject needs to re-run
+    conversion.
+    """
+
+    @staticmethod
+    def check_conversion(
+        args: argparse.Namespace,
+        conv_folder: Path,
+    ) -> bool:
+        """Checks if conversion needs to be run based on user input and the file structure.
+        If the user wants to run conv, we run it. Otherwise, check that all the necessary files
```

Comment:
can we in parenthesis add how the user forces it to re-run "If the user wants to run conv (HOW) ..."?